stable_baselines3/common/base_class.py

```diff
@@ -161,6 +163,15 @@ def __init__(
         if env is not None:
             if isinstance(env, str):
                 if create_eval_env:
+                    warnings.warn(
+                        "The parameter `create_eval_env` is deprecated and will be removed in the future. "
+                        "Please use `EvalCallback` or a custom Callback instead.",
+                        DeprecationWarning,
+                        # By setting the `stacklevel` we refer to the initial caller of the deprecated feature.
+                        # This causes the the `DepricationWarning` to not be ignored and to be shown to the user. See
+                        # https://github.com/DLR-RM/stable-baselines3/pull/1082#discussion_r989842855 for more details.
+                        stacklevel=4,
+                    )
                     self.eval_env = maybe_make_env(env, self.verbose)
```

Comment:
From https://github.com/DLR-RM/stable-baselines3/issues/925#issuecomment-1251488507, I understand that this feature should be removed. Shouldn't this line be deleted?

Reply:
> Shouldn't this line be deleted?

only after the next release, no?
First deprecate and then remove?

Reply:
I understood that we would first deprecate the parameters and then remove them in a subsequent release (As it would be breaking changes for those who use them).

Reply:
ok for me